src/resources/communityContributionCompany.ts

```diff
@@ -0,0 +1,91 @@
+import { ResourceIF } from '@/interfaces'
+import { NameRequestTypes, RouteNames } from '@/enums'
+import { CorpTypeCd, GetCorpFullDescription } from '@bcrs-shared-components/corp-type-module'
+
+export const CommunityContributionCompanyResource: ResourceIF = {
+  entityType: CorpTypeCd.BC_CCC,
+  displayName: GetCorpFullDescription(CorpTypeCd.BC_CCC),
+  title: 'Community Contribution Company Statement',
+  description: null,
+  statement: null,
+  nameRequestType: NameRequestTypes.CC,
+  steps: [
```

Comment:
What do you think of extracting these steps into a common file used by BEN/ULC/etc ?

Is it better to be verbose/explicit here, or is it better to keep common things together?

Reply:
I considered templating them but wasn't sure. 

If we can say this is default template then so be it, but i wasn't really sure what the future holds, so i just defined them explicitly on the entity config, in the event something does change during development here.

Reply:
But i agree, why don't we start from a template and break them out if required. 👍 
Will update.

Reply:
I like that there are separate resource files for each business type. I'm just wondering if we should pull out the steps (which are common here, right) for these companies only?

One possible downside is that this introduces the concept of "default" or "fallback" steps, and we might eventually be tempted to say "use defaults except for ...", and that will get complicated fast.

Maybe with a good name for the steps common to BEN/ULC/etc, this will make sense.

I think a lot will make sense as we work through this. As long as we're intentional about keeping complexity low (at the cost of duplication if necessary), we should be OK 🤞

Reply:
Thats the general idea with the duplication, was to keep it simple to start.
My initial gut feeling was to create a template for these 4 entities (BEN, CCC, ULC, BC), who all share the same steps.
I guess i don't trust that design won't change something along the way, so this way we can update them individually, although i think i will go with the template to start and if changes happen, we can break them out into there own.

Reply:
OK, we're on the same page.

Go whichever way you want here.

Reply:
kk, Updated to a single template for now! 

I couldn't think of a better name than BaseStepsTemplate, but i did add to the comments which entities it applies too.

Reply:
OK but let's rename that file/structure soon. It is not a template or base steps (as I don't want us to ever extend it). It is the common steps for BEN/BC/ULC/CCC.